src/Xunit.StaFact.Desktop/Sdk/DispatcherSynchronizationContextAdapter.cs

```diff
@@ -41,5 +41,11 @@ internal override void Run(Func<Task> work)
             this.PumpTill(task);
             task.GetAwaiter().GetResult();
         }
+
+        internal override void Cleanup()
+        {
+            Dispatcher.CurrentDispatcher.InvokeShutdown();
```

Comment:
@bradphelan  actually I am not so sure.. I see that all WPF tests start their own thread created in the RunOnSTA method(https://github.com/ilengyel/Xunit.StaFact/blob/4e4f0c35cfce030a08df22a958b95c10c3e0b398/src/Xunit.StaFact.Shared/Sdk/UITestInvoker.cs#L83), and have their own dispatcher.. which I assume is Dispatcher.CurrentDispatcher